### DISCUSSION TITLE
Create Data Platform project with SQLite DB

### DISCUSSION THREAD

**Discussion by Apchibudzdravij** [prisma/prisma#19023] [Q&A] [answered]
Is it possible to create a project in Data Platform with SQLite database? If yes, what shall I type in the "Connection string" field?

**Comment by nurul3101** [comment, marked as answer]
Hello @Apchibudzdravij 👋 

No, it's not possible to create a project in [Data Platform](https://cloud.prisma.io/) with a SQLite database. I would recommend you to try out data platform with any other database like PostgreSQL, MySQL, CockroachDB or MongoDB.
